exercise3/calculateDiscount.js

```diff
@@ -0,0 +1,5 @@
+function calculateDiscount(totalAmount) {
+  let disc = 0;
+  if (totalAmount > 100) return (disc = totalAmount * 0.1);
+  return (disc = totalAmount * 0.05);
+}
```

Comment:
- I love the improvements you have made to this code, However, do you mind using the ternary operator to reduce the lines of code since with `ternary operator` you can write the whole code in one line, and the `disc` variable I feel it is redundant.